src/en/space-station-14/round-flow/proposals/revolutionaries-codeword-rework.md

```diff
@@ -0,0 +1,271 @@
+
+The Captain is also unique in that they always have 100 LHP and can never go below this. This makes the Captain unconvertable to the Revolution, and they are the only Command on the station with this property. CentComm agents and other Admeme should be similarly unconvertable by default.
+
+Unlike normal damage, LHP can only be dealt/healed to an individual who is alive and not in crit. 
+
+### Revolutionary Codewords & LHP Damage
+
+Revolutionary codewords is the main way a Rev convert crew, and is the primary conversion method the gamemode is balanced around. When becoming a Revolutionary (Headrev or converted) you are provided with a number of codewords that when spoken deals LHP damage to anyone hearing it. These codewords are unique to each Headrev and are passed on to whatever Revolutionary they convert, and any further conversions down the chain get the same codewords passed on. 
+
+Codewords come in three levels; Low, Mid and High Conversion words, with **5** codewords given in each level.
+  - Low Conversion words are meant to be easy to incorporate into normal speech. There should be so many possible low conversion words that can be rolled that it becomes effectively impossible to metagame. Examples: "love", "hello", "great", "hate".
+  - Mid Conversion words deal a decent amount of LHP damage but can be noticed if one pays attention. These would be at the risk of metagaming due to being flavored towards revolutionary speech, so it would be on the rev player to incorporate them well into conversations. Examples: "disrupt", "rise up", "join".
+  - High Conversion words are very clearly revolutionary and deal high LHP damage, *and is able to convert a crewmember in the Convertable state. They should be mostly useful when the revolution is in full swing, but risky for a non-convertable crewmember to overhear. Examples: "viva", "communism", "comrade".
+
+Dealing damage with codeword has a **10** second cooldown, to avoid simply spamming the codewords. Damage is also split among the crew who hear it, making speech directed to a single individual more effective at targeted conversion, while multiple Revolutionaries can shout it out to crowds without the conversion happening to fast. Using multiple codewords in one sentence does not do anything; it simply uses the highest-tier codeword instead.
+
+Codewords are proposed to deal the following damage:
+  - Low: **10** LHP
+  - Mid: **20** LHP
+  - High: **30** LHP, and converts any crew in the Convertable state.
+
+Codewords are able to deal damage even when communicated across different mediums, however the conversion strength is reduced to compensate for how easy it may be to do so:
+
+- A Revolutionary speaking a codeword through a headset radio deals **0.2**x LHP damage. (Note that crew passively heal LHP, as described in the Deconversion section)
```

Comment:
So the most obvious problem with this is that this falls over as soon as "deaf" is added as a trait. But otherwise it makes talking and hanging around near people a weapon for Revs. So if Revs is confirmed... you just take off your headset and stop allowing people to talk?

Reply:
If we make it so that you can't talk in space, you could actually neuter Revs by just spacing the station.

Reply:
For a confirmed revolution, popping out your headset key is an intended counter.

If you're deaf then, sure, conversion will be harder. If lip reading/sign language is a thing then this can be mitigated, and there are other methods of dealing LHP that don't involve speech.